src/utils/fetchJson.ts

```diff
@@ -1,10 +1,10 @@
 import { HttpVerbs } from "src/@types/HttpVerbs";
 
-export type Methods = Exclude<HttpVerbs, HttpVerbs.GET>;
+export type Method = Exclude<HttpVerbs, HttpVerbs.GET>;
```

Comment:
As we can't merge you PR because of gh actions, just a comment:
If you need to export a type (i.e. it is used in more than one place), you should put the type in its own file, under `@types` 🙂.

Reply:
OK done! c7a43df

Reply:
Pro 💪